guides/common/modules/con_scenarios-for-foreman-on-aws.adoc

```diff
@@ -0,0 +1,84 @@
+[id="scenarios-for-{project-context}-on-aws"]
+= Scenarios for {Project} on AWS
+
+There are several deployment scenarios for {Project} on Amazon Web Services (AWS):
+
+* One-region setup
+* Connecting on-premises and AWS region
+* Connecting different regions
+
+.Scenario 1: One-region setup
+ifdef::foreman-el,foreman-deb,katello[]
+image::common/aws-one-region-setup.png[One-region setup]
```

Comment:
You should probably first describe the scenario and then show the image, the way it's done for the other scenarios.